### PR TITLE
Hopo Berry

### PR DESCRIPTION
**Pull request by Qwertypop04**
## Description
Adds Hopo Berry from Pokemon Legends Arceus, with the mutation method commented out so as to not yet be obtainable

## Motivation and Context
Part of Hisui #4128 

## Types of changes
- New content

### CODE REVIEW

File: src/scripts/farming/Farming.ts [hunk 1]
```diff
@@ -1220,6 +1220,21 @@ class Farming implements Feature {
...
+            1,
+            0,
+            15000,
+            25,
+            [15, 40, 35, 30, 25],
+            49.3,
+            BerryColor.Gold,
+            5.8,
+            BerryFirmness.Very_Soft,
+            ['A truly mythical Berry. It is said to have been first created by a god.']
```
Comment (osherwott) [inline src/scripts/farming/Farming.ts:1236]:
```suggestion
            ['A hardy Berry native to harsh northern lands. If a wild Pokémon eats one of these Berries, that Pokémon's reactions will be dulled.']
```
Most of our berry descriptions use official Tag information, even if they don't make sense to pokeclicker mechanics. I've added one sentence from PLA and one generic descriptor for flavor text. We should also avoid theistic references lest we offend someone or are accused of blasphemy.

**Review by osherwott — COMMENTED** [review]
Mind taking a look at these suggestions?

File: src/scripts/farming/Farming.ts [hunk 2]
```diff
@@ -1220,6 +1220,21 @@ class Farming implements Feature {
...
+
+        this.berryData[BerryType.Hopo] = new Berry(
+            BerryType.Hopo,
+            [10800, 21600, 43200, 86400, 604800],
+            1,
+            0,
+            15000,
+            25,
+            [15, 40, 35, 30, 25],
+            49.3,
```
[the remark this answers is not in the document]
Comment (Qwertypop04) [reply src/scripts/farming/Farming.ts]: RIP reference. Fixed

File: src/scripts/farming/Farming.ts (hunk 1 above)
Comment (Qwertypop04) [reply src/scripts/farming/Farming.ts:1236]: Altered to make the reference to Arceus more clear

**Comment by osherwott** [comment]
For future reference:
- Size and Firmness are based off of the hops plant, which is the likely inspiration for the berry
- Flavor profile is based off of
  - its pokeclicker status (at least one value of 40)
  - the description of Jubilife Muffins, in which two Hopo Berries are combined with one Razz Berry to create a primarily dry, non-"mellow" flavor (multiple relatively high flavors)

**Review by Jaaslet — APPROVED** [review]
Looks good. Thanks for the PR

**Pull request merged**